src/rotation.rs

```diff
@@ -0,0 +1,59 @@
+use vector::Vector3;
+
+
+/// Euler rotation angles in 3D space.
```

Comment:
IMO a note about the sequence in which the different rotations are applied should be added.
e.g `cgmath` uses xyz (tait–bryan) iirc, but other libraries might use a different ordering.